source/quickguides/index.rst

```diff
@@ -8,5 +8,5 @@ Quick Guides
    quick-guide-general
```

Comment:
Do we even want the general quick guide? To show proper code, we need to know what machine we're on. We could just have the three machine-specific quick guides and then have a general guide to list `obs_path`, `test_data_path`, `html_path`, and `web_address` for the remaining machines.

Reply:
I agree; we don't really need a general quick guide anymore. If a user wants to run on a different machine, they can probably extrapolate from one of the existing guide.